src/components/ContentMapping/ContentMapping.ts

```diff
@@ -5,6 +5,10 @@ import { Stub, StubEditor } from '../ContentWidgets/_Stub/Stub';
 import { ContentSingularData } from '../_data/ContentSingularData';
 import { ExampleImage } from '../ContentWidgets/ExampleImage/ExampleImage';
 import { ExampleImageEditor } from '../ContentWidgets/ExampleImage/ExampleImageEditor';
+import { TabView } from '../ContentWidgets/TabView/TabView';
```

Comment:
Merge in the master branch (or if you aren't comfortable with fixing all merge conflicts then I can do it)